Makefile

```diff
@@ -131,7 +131,8 @@ docker-image-runtime:
 	cd contrib/packaging/docker && docker build -t "cilium/cilium-runtime:$(UTC_DATE)" -f Dockerfile.runtime .
 
 docker-image-builder:
-	cd envoy && docker build -t "cilium/cilium-builder:$(UTC_DATE)" .
+	cd envoy && docker build -t "quay.io/cilium/cilium-builder:$(UTC_DATE)" .
+	@echo "docker push quay.io/cilium/cilium-builder:$(UTC_DATE)"
```

Comment:
Is this intended as an instruction? It might look like an actual line being executed.

Reply:
Yes it’s an instruction. I followed the same pattern as the other target that builds a Docker image.